core/src/test/java/org/jboss/jandex/test/BasicTestCase.java

```diff
@@ -468,6 +474,54 @@ public void testNullClass() throws IOException {
         indexer.indexClass(null);
     }
 
+    @Test
+    public void testRuntimeInvisiblePresentInV11() throws IOException {
+        @RuntimeInvisible(placement = "class")
+        class Target {
+            @SuppressWarnings("unused")
+            void execute(@RuntimeInvisible(placement = "arg") String arg) {
+            }
+        }
+
+        assertEquals(0, Target.class.getDeclaredAnnotations().length);
+        assertEquals(0, Target.class.getDeclaredMethods()[0].getDeclaredAnnotations().length);
```

Comment:
The `@RuntimeInvisible` annotation is on a parameter, so shouldn't this be more like `Target.class.getDeclaredMethods()[0].getParameters()[0].getDeclaredAnnotations()`?

Reply:
Also, could you add a runtime-invisible type annotation? E.g. `void execute(@RuntimeInvisible(placement = "arg") List<@RuntimeInvisible(placement = "type use") String> arg) {}`.

Reply:
Fixed and added a "type use" case

Reply:
Note that the test uses Java 8 features now - which I assume is fine for 3.0.

Reply:
Yes, 3.0 will be based on 8, so that's fine.